petab/visualize/helper_functions.py

```diff
@@ -40,9 +40,14 @@ def import_from_files(
                                      pd.DataFrame, pd.DataFrame]:
     """
     Helper function for plotting data and simulations, which imports data
-    from PEtab files.
+    from PEtab files. If path to the visualization file is not provided, the
```

Comment:
```suggestion
    from PEtab files. If `visualization_file_path` is not provided, the
```
can one make visualization_file_path optional too? above, -> `visualization_file_path: str = None`, and move one down. and below, change the check for `visualization_file_path != ''` to ``visualization_file_path not in ['', None]``

Reply:
maybe just `if visualization_file_path` instead of `visualization_file_path != ''`